src/RestController.php

```diff
@@ -0,0 +1,82 @@
+<?php # -*- coding: utf-8 -*-
+declare(strict_types=1);
+
+namespace MultisiteGlobalMedia;
+
+use WP_REST_Attachments_Controller;
+
+/**
+ * Class Rest
+ */
+class RestController extends WP_REST_Attachments_Controller
+{
+    use Helper;
+
+    /**
+     * @var Site
+     */
+    private $site;
+
+    /**
+     * @var SiteSwitcher
+     */
+    private $siteSwitcher;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function __construct($post_type) // phpcs:ignore
```

Comment:
Please enhance the comment to follow form: `phpcs:disable Rule and reason` not only `phpcs:ignore`. Please see also your other phpcs comment, so that developers read about the reason. Thx.
so it's overriding the parent method but the phpcs:ignore does not give us any info about why the name is in snake_case nor the ignore of the parameter type hints